src/event-stream/event-server.ts

```diff
@@ -931,174 +928,145 @@ export async function startEventServer(opts: {
     eventHost = hostname;
   }
 
-  const app = express();
+  const bodyLimit = 1_000_000 * 500; // 500MB body limit
```

Comment:
Another value we should set as configurable in the next PR

Reply:
https://github.com/hirosystems/stacks-blockchain-api/issues/2055